fixup-mountpoints

```diff
@@ -640,6 +640,69 @@ case "$DEVICE" in
             -e 's block/bootdevice/by-name/userdata mmcblk0p46 ' \
            "$@"
         ;;
+	
+	"oneplus3")
+        sed -i \
+            -e 's block/bootdevice/by-name/LOGO block/sde17 ' \
```

Comment:
You don't have to `block/sde17` but just `sde17` here..

So this line becomes, e,g.
`
-e 's block/bootdevice/by-name/LOGO sde17 ' \
`